source/crud/tests/v2/findOneAndDelete-hint-error.yml

```diff
@@ -0,0 +1,52 @@
+runOn:
+  - { maxServerVersion: "4.1.10" }
```

Comment:
> Looks good, but can we add negative tests that prove drivers error when hint is specified when running on older server versions?

@jyemin: This is now done. The three new tests are copies of the original test with `result` replaced with `error: true` and `outcome` changed to assert the original collection data was unchanged. I've used 4.1.10 as a max server version here since that's when I know the server started raising its own error for unrecognized `findAndModify` options ([SERVER-40005](https://jira.mongodb.org/browse/SERVER-40005)), but there's no reason this couldn't be changed to 4.2.0 since drivers will likely use wire versions to determine whether a client-side error should be raised. AFAIK, that's why we don't mention specific patch versions in the spec text, but I like doing so in the tests (it definitely makes sense with min server version to allow testing before stable releases).

Reply:
While reviewing https://github.com/mongodb/specifications/pull/692 I realized that I should have actually used 4.1.9 here since `maxServerVersion` is _inclusive_. Since we're testing for client-side errors, this should allow execution on a server up to, but not including, the version when server-side support was introduced. I'll make the change.